docs/kubernetes/cloud-config/node-config-template.yaml

```diff
@@ -0,0 +1,126 @@
+#cloud-config
+---
+write_files:
+  # Network config file for the Calico CNI plugin.
+  - path: /etc/cni/net.d/10-calico.conf 
+    owner: root
+    permissions: 0755
+    content: |
+      {
+          "name": "calico-k8s-network",
+          "type": "calico",
+          "etcd_authority": "172.18.18.101:2379",
+          "log_level": "info",
+          "ipam": {
+              "type": "calico-ipam"
+          }
+      }
+
+  # Kubeconfig file. 
```

Comment:
This is used by the DNS service (and potentially other services that might be deployed) to access the API.